skbio/stats/distance/_base.py

```diff
@@ -715,28 +767,31 @@ def from_iterable(cls, iterable, metric, key=None, keys=None):
 
         Notes
         -----
-        Symmetry and hollowness are assumed when calculating the distances via
-        `metric`. Therefore, distances are only computed for the strictly
+        When `validate=True`, distances are only computed for the strictly
```

Comment:
Remove this "Notes" section since the details are covered above in the description for `validate`.